egs2/aishell4/asr1/local/data.sh

```diff
@@ -0,0 +1,298 @@
+#!/usr/bin/env bash
+
+. ./path.sh || exit 1;
+. ./cmd.sh || exit 1;
+. ./db.sh || exit 1;
+
+
+log() {
+    local fname=${BASH_SOURCE[1]##*/}
+    echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+stage=0       # start from 0 if you need to start from data preparation
+stop_stage=100
+FOLDER=git_aishell
+
+ . utils/parse_options.sh || exit 1;
+
+mkdir ${AISHELL4}
```

Comment:
Please add `-p` option so as not to interrupt the process if the dir is already there